### ISSUE TITLE
quests with flags

### ISSUE THREAD

**Issue by yellow-pillow** [GC-spigot/battle-pass#107]
tasks related to flags do not work

(tried to create a task to put a white flag - does not work, decided to make a quest to scrap white flag - also does not work)

**Comment by GC-spigot** [comment]
There's literally formats for issues, it's not hard to follow them. Now I don't have examples or any information, I don't know if it's an issue on your end, specific jar or else.

**Comment by yellow-pillow** [comment]
** Error description**
block_place with white_banner  does not work NORMALLY

**Replay**.
try making a white flag task, put it up and 90% of the time it won't work because it only works when turned to a certain side (video)

**Screenshots/video**
https://drive.google.com/file/d/1qSARa6aVTNx53ane8D1lwzJxQuoGLTM2/view?usp=sharing


**Server info (please fill in the following information):**.
- Server version: [ssspigot 1.16.5].
- Plugin version: [latest 3.5.18].

**addition**.
see video to the end! i showed a bug there, most likely this is the main mistake!

**Comment by yellow-pillow** [comment]
MORE BAG!

if there is a block_place jack_o_lantern, it only works in one direction -_- why?

**Comment by yellow-pillow** [comment]
I also recommend checking the other pumpkins, heads, there is a feeling that the armor racks, bell, turtle eggs and other units with different positions will work STRANGELY 0_0

**Comment by GC-spigot** [comment]
Fixed
